modules/core/src/main/scala/lucuma/odb/api/model/gmos/longslit/GmosLongslitMath.scala

```diff
@@ -98,7 +98,7 @@ object GmosNorthLongslitMath extends GmosSiteSpecificLongslitMath {
   override def site: Site = Site.GN
 
   def effectiveSlitWidth(fpu: GmosNorthFpu, p: SourceProfile, iq: ImageQuality): Angle =
-    fpu.slitWidth.getOrElse(IfuSlitWidth) min effectiveSize(p, iq)
```

Comment:
I'm not sure if the changes in this file are correct. There were changes in lucuma-core that broke this code, and these changes compiled...

Reply:
Yes thanks, that looks right to me.